src/Indicator.vala

```diff
@@ -125,11 +126,24 @@ public class DateTime.Indicator : Wingpanel.Indicator {
             menuitem_label.wrap_mode = Pango.WrapMode.WORD_CHAR;
             menuitem_label.xalign = 0;
 
-            var menuitem_box = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 6);
+            var menuitem_times = new Gtk.Label ("");
+            menuitem_times.set_markup (e.get_event_times ());
```

Comment:
In Calendar we use both `<small>` and `DIM_LABEL`. I think it probably makes sense to do that here as well
```suggestion
            menuitem_times.set_markup ("<small>%s</small>".printf (e.get_event_times ()));
```